.neutrinorc.js

```diff
@@ -79,6 +168,10 @@ module.exports = {
           bindings: 'bindings'
         }));
 
+      // TODO: Why is this needed?
+      neutrino.config.resolve.modules.add(MODULES);
```

Comment:
I would recommend not adding this. If not having it causes an error, could you include it in the code comment? (And I'm happy to take a look)